.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,16 +1,6 @@
-[TIP]:  # ( Provide a general summary of your changes in the title above ^^ )
-
-## Type of change
-[NOTE]: # ( Please remove all lines which don't apply. )
-- ✅ Bug fix (non-breaking change which fixes an issue)
-- ✅ Refactor (significant modification to existing code)
-- ✅ New feature (non-breaking change which adds functionality)
-- ✅ Breaking change (fix or feature that would cause existing functionality to change)
-- ✅ Documentation (non-code change)
-
-## Description and Context
+## Summary
```

Comment:
This heading should be removed. Github automatically inserts the commit message at the top, which is basically the summary.

Reply:
I wish GitHub would skip over the first heading label or allow for a fill-in tag like `<commitmsg>`